src/common/Card/helper.ts

```diff
@@ -0,0 +1,41 @@
+// just a list of all months in order
+const months = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+// Input: Date object, duration (minutes)
+// Output: Formatted string for start time and end time of an event
```

Comment:
It would be nicer if we followed the JSDoc specification for documentation comments. Check out https://jsdoc.app/ for more info. There's also a vscode extension that can automatically create jsdocs for you, you could find it by searching

Reply:
my bad, donezo :+1: